app/models/wise_transfer.rb

```diff
@@ -0,0 +1,176 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: wise_transfers
+#
+#  id                            :bigint           not null, primary key
+#  aasm_state                    :string
+#  account_number_bidx           :string
+#  account_number_ciphertext     :string
+#  address_city                  :string
+#  address_line1                 :string
+#  address_line2                 :string
+#  address_postal_code           :string
+#  address_state                 :string
+#  amount_cents                  :integer          not null
+#  approved_at                   :datetime
+#  bank_name                     :string
+#  branch_number_bidx            :string
+#  branch_number_ciphertext      :string
+#  currency                      :string           not null
+#  institution_number_bidx       :string
+#  institution_number_ciphertext :string
```

Comment:
I assume these are being stored as separate columns to encrypt them. Can we try directly encrypting `recipient_information`? I'd like to avoid us having to add new columns as we add support for new countries. See the `PaymentRecipient` as an example of this.